core/domain/classifier_services.py

```diff
@@ -327,17 +311,17 @@ def save_classifier_training_job(algorithm_id, exp_id, exp_version,
             classifier.
 
     Returns:
-        job_id: str. ID of the classifier training job.
+        job_id: str or None. ID of the classifier training job.
```

Comment:
@seanlip I'll make the changes here as well after we decide on a solution for the above comment.